openai-core/src/commonMain/kotlin/com.aallam.openai.azure.api/filtering/ContentFilterSeverity.kt

```diff
@@ -0,0 +1,44 @@
+package com.aallam.openai.azure.api.filtering
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Ratings for the intensity and risk level of harmful content.
+ */
+@Serializable
+public enum class ContentFilterSeverity(@Serializable public val value: String) {
```

Comment:
Could you please change `enum` to a `value class` with `companion object` values? This pattern is utilized in the library because it allows the API to evolve without limiting users in that scenario.